package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fin-hypergrid",
-  "version": "2.0.2",
+  "version": "2.1.0",
```

Comment:
Shouldn't we change this in a separate commit right before we cut a version?

Reply:
@nmichaud sure, i'll pull it out